src/genie/libs/parser/iosxe/show_vtp.py

```diff
@@ -96,13 +96,13 @@ class ShowVtpStatusSchema(MetaParser):
                     'vlan': {
                         'enabled': bool,
                         'operating_mode': str,
-                        'maximum_vlans': int,
+                        Optional('maximum_vlans'): int,
```

Comment:
Hi @TopSecretAtWork ,

I checked our code:
https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxe/show_vtp.py#L93
where both keys are already optional.

Could you please check if you're using the latest version?

> pyats -- 20.4
> genie.libs.parser -- 20.4.1b2


Thanks,
Irene

Reply:
Hi Irene,

I'm sorry, I have expressed myself unclear.

I changed the values in the Optional('feature') -> 'vlan' part of the schema.

https://github.com/CiscoTestAutomation/genieparser/blob/7704b0adf9997263c32c71384598a6b2787a2c37/src/genie/libs/parser/iosxe/show_vtp.py#L99

https://github.com/CiscoTestAutomation/genieparser/blob/7704b0adf9997263c32c71384598a6b2787a2c37/src/genie/libs/parser/iosxe/show_vtp.py#L105